modules/api-svc/src/lib/model/common/Enums.js

```diff
@@ -33,7 +36,18 @@ const TransactionRequestStateEnum = {
     REJECTED: 'REJECTED',
 };
 
+const Directions = Object.freeze({
+    INBOUND: 'INBOUND',
+    OUTBOUND: 'OUTBOUND',
+});
+
+const CacheKeyPrefixes = Object.freeze({
+    FX_QUOTE: 'fxQuote_in'
```

Comment:
Do we need a separate prefix for FX cache? Lets discuss this

Reply:
I thought, we should save _quotes_ and _fxQuotes_ with different keys (coz they are different "entities"). But I might be wrong. Let's discuss

Reply:
As we have a condition and calling different functions for save to cache, lets use separate keys for now.